pkg/protocol/sybilprotection/sybilprotectionv1/performance/rewards.go

```diff
@@ -65,19 +56,49 @@ func (t *Tracker) ValidatorReward(validatorID iotago.AccountID, stakeAmount iota
 			return 0, 0, 0, ierrors.Errorf("pool stats for epoch %d and validator accountID %s are nil", epochIndex, validatorID)
 		}
 
-		unDecayedEpochRewards := uint64(rewardsForAccountInEpoch.FixedCost) +
-			decreaseAccuracy(poolStats.ProfitMargin*uint64(rewardsForAccountInEpoch.PoolRewards), profitMarginExponent) +
-			decreaseAccuracy(increasedAccuracyComplement(poolStats.ProfitMargin, profitMarginExponent)*uint64(rewardsForAccountInEpoch.PoolRewards), profitMarginExponent)*
-				uint64(stakeAmount)/
-				uint64(rewardsForAccountInEpoch.PoolStake)
+		// all rewards go to delegators, as the fixed cost was too high
+		if rewardsForAccountInEpoch.PoolRewards < rewardsForAccountInEpoch.FixedCost {
+			return 0, 0, 0, nil
```

Comment:
I always have problems testing things in this repo, but isn't this `return` zeroing the validator rewards for the whole loop (from `epochStart` to `epochEnd`)?

Reply:
Yes, this would return 0 validator rewards. Is this correct?

Reply:
I thought about just zeroing the reward for the epoch in question (the one where `rewardsForAccountInEpoch.PoolRewards`<`rewardsForAccountInEpoch.FixedCost`)

I have to think a little bit about whether this is a problem or not, maybe that's too harsh, idk...

Reply:
I thought that I had already commented this somewhere, but maybe I commented in the wrong place or something. Where is this `rewardsForAccountInEpoch.PoolRewards` being set? because the `poolReward` function returns the pool reward value after subtracting the fixed cost (see line 244) and then this check doesn't make sense.

Reply:
(I can't find where you update the `PoolRewards` struct)

Reply:
Also, I suggested (somewhere :p) to just not add any value for this single epoch to the validator rewards, instead of zeroing the rewards for the whole period